packages/server/src/main/java/software/uncharted/terarium/hmiserver/service/tasks/ConfigureModelResponseHandler.java

```diff
@@ -78,41 +76,22 @@ public TaskResponse onSuccess(final TaskResponse resp) {
 				modelCopy.setId(model.getId());
 
 				// Map the parameters values to the model
-				final ArrayNode gollmExtractions = objectMapper.createArrayNode();
 				if (condition.has("parameters")) {
 					final List<ModelParameter> modelParameters =
 							ScenarioExtraction.getModelParameters(condition.get("parameters"), modelCopy);
-					if (modelCopy.isRegnet()) {
-						modelCopy
-								.getModel()
-								.put("parameters", objectMapper.convertValue(modelParameters, JsonNode.class));
-					}
-					final ArrayNode parameters = condition.get("parameters").deepCopy();
-					gollmExtractions.addAll(parameters);
+					modelCopy.getSemantics().getOde().setParameters(modelParameters);
 				}
 
 				// Map the initials values to the model
 				if (condition.has("initials")) {
 					final List<Initial> modelInitials =
 							ScenarioExtraction.getModelInitials(condition.get("initials"), modelCopy);
-					if (modelCopy.isRegnet()) {
-						modelCopy.getModel().put("initials", objectMapper.convertValue(modelInitials, JsonNode.class));
-					}
-					final ArrayNode initials = condition.get("initials").deepCopy();
-					gollmExtractions.addAll(initials);
+					modelCopy.getSemantics().getOde().setInitials(modelInitials);
 				}
 
-				// Set the all the GoLLM extractions into the model metadata
-				// FIXME - It is not what we should do, this is a hack for the March 2024
-				// Evaluation
-				if (model.getMetadata() == null) {
-					model.setMetadata(new ModelMetadata());
-				}
-				model.getMetadata().setGollmExtractions(gollmExtractions);
```

Comment:
Do we not want gollm extractions anymore?

Reply:
They were used to create the Model configuration when the asset did not exist.
Now that we have a model configuration asset, those are not needed anymore.